src/rules/memberOrderingRule.ts

```diff
@@ -17,16 +17,55 @@
 import * as Lint from "../lint";
 import * as ts from "typescript";
 
+/* start old options */
 const OPTION_VARIABLES_BEFORE_FUNCTIONS = "variables-before-functions";
 const OPTION_STATIC_BEFORE_INSTANCE = "static-before-instance";
 const OPTION_PUBLIC_BEFORE_PRIVATE = "public-before-private";
+/* end old options */
+
+/* start new options */
+const OPTION_ORDER = "order";
+const PRESET_ORDERS: { [preset: string]: string[] } = {
```

Comment:
Looks like the instance-sandwich will suit me the best but as long as I can order the rules by myself I'm totally fine with that.